### DISCUSSION TITLE
Call to undefined method Livewire\LivewireManager::originalMethod()

### DISCUSSION THREAD

**Discussion by ergagandeep** [livewire/livewire#5346] [1. Help]
Can someone help on Livewire issue?
This Laravel project is working good on local machine and faced the problem on Server. The virtual host is pointed upto "public" folder.
https://d41e8a49-25ed-4507-b65b-cdc9589c2f31.clouding.host/login
Email: test@test.com
Password: 12345

Thanks for the help

**Comment by richardkeep** [comment]
the link is not accessible

**Comment by digitlimit** [comment]
Run 
```
php artisan clear-compiled
php artisan optimize:clear
```

**Reply by ilramdhan** [reply]
> Run
> 
> ```
> php artisan clear-compiled
> php artisan optimize:clear
> ```

is it work?

**Reply by stephenr85** [reply]
This did work for me. I started seeing the error after updating my dev-tools package. I am however, still getting an error at the /admin/devtools/model/search endpoint that I'm trying to hunt down...

**Reply by lokesh-zersys** [reply]
i was upgrading from livewire v2 to v3, and got this issue, running the above commands fixed it.

**Comment by romanmiranda** [comment]
After a Livewire update run:

`php artisan view:clear`

To clear the view cache.
